contrib/ansible/roles/kubernetes/tasks/gen_tokens.yml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
You forgot to copy the script to the host. My run fails becasue the /kube-gen-token.sh is not there.